opentok/opentok.py

```diff
@@ -1422,30 +1424,45 @@ def __init__(
     
 
 
-    def mute(self, session_id: str, stream_id: str= "", options: dict = {}) -> requests.Response:
+    def mute(self, 
```

Comment:
@geekchick Let's change this to two methods: `mute_all(session_id, active, excludedStreamIds)` and `mute_stream(session_id, stream_id`. 

What is the `data` parameter? Can we remove it?

Reply:
Hi @jeffswartz the data parameter is the body or payload for the request. It gets defined as a dictionary in the method signature and used in the code to populate the request body.